ethereum/jsonrpc/src/test/java/tech/pegasys/pantheon/ethereum/jsonrpc/websocket/subscription/syncing/SyncingSubscriptionServiceTest.java

```diff
@@ -38,6 +40,7 @@
   @Mock private SubscriptionManager subscriptionManager;
   @Mock private Synchronizer synchronizer;
   private SyncStatusListener syncStatusListener;
+  private final Vertx vertx = Vertx.vertx();
```

Comment:
You need to close `vertx` in a tear down, but see above for a way to avoid having these classes use vertx directly.  Then you could just have a `when(subscriptionManager.notifySubscribers(...)).thenAnswer(...)` where the answer would invoke the callback immediately so you don't need to deal with multiple threads in the tests.